packages/editor-ui/src/components/BinaryDataDisplay.vue

```diff
@@ -111,19 +111,5 @@ export default mixins(nodeHelpers, restApi).extend({
 			height: 100%;
 		}
 	}
-
-	.binary-data {
```

Comment:
this is also defined [here](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/BinaryDataDisplayEmbed.vue#L82)